contribution-guide.md

```diff
@@ -0,0 +1,143 @@
+---
```

Comment:
@NipunaMarcus Instead of duplicating the content in two places (https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md and here), is there a way to generate b.io page ?

Reply:
As I found we can use submodule to get content from other repos. https://help.github.com/en/github/working-with-github-pages/using-submodules-with-github-pages

But its okay to have these type of duplicate docs in site coz we have to anyway change the markdown to have a jekyll layout to match other pages.

Reply:
+1 for duplicates then as these contents are not frequently changing.